### PR TITLE
Add render_embed in to Page.py

### PR DESCRIPTION
**Pull request by lakdred**
本次 PR 内容，
Add render_embed in to Page.py

### CODE REVIEW

File: pyecharts/charts/composite_charts/page.py [hunk 1]
```diff
@@ -89,6 +91,21 @@ def render_notebook(self):
...
 
+    def render_embed(
+        self,
+        template_name: str = "simple_page.html",
+        env: Optional[Environment] = None,
+    ):
+        self.js_functions=OrderedSet()
+        for c in self:
+            self.json_contents = c.dump_options()
+            for item in c.js_functions.items:
```
Comment (chenjiandongx) [inline pyecharts/charts/composite_charts/page.py]: page 不需要 js_functions，js_functions 由每个 chart 自行提供

File: pyecharts/charts/basic_charts/radar.py [hunk 2]
```diff
@@ -25,9 +26,18 @@ def add_schema(
         splitarea_opt: Union[opts.SplitAreaOpts, dict] = opts.SplitAreaOpts(),
         axisline_opt: Union[opts.AxisLineOpts, dict] = opts.AxisLineOpts(),
     ):
+        if isinstance(textstyle_opts, opts.TextStyleOpts):
```
Comment (chfw) [inline pyecharts/charts/basic_charts/radar.py]: I think, no long this is needed. https://github.com/pyecharts/pyecharts/pull/1058 has removed such a need!

**Comment by chfw** [comment]
Again, unit test?

**Comment by chenjiandongx** [comment]
请统一好 PR，一个 feature 开了三个 PR？

**Pull request closed without merging**